### ISSUE TITLE
Fix accessibility errors on the SprkCards on the SparkDesignSystem.com homepage

### ISSUE THREAD

**Issue by bobwise** [sparkdesignsystem/spark-design-system#3739]
1. Cards should have a semantically correct header level (or no header level if appropriate). Requires #3300
2. CTAs in cards should be semantic links styled like buttons. Requires #3737
3. Cards should render without loss of information or functionality, and without requiring horizontal scrolling at a width of 320 px. See [WCAG 1.4.10 Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html).
